swri_transform_util/include/swri_transform_util/local_xy_util.h

```diff
@@ -168,6 +169,13 @@ namespace swri_transform_util
     std::string Frame() const { return frame_; }
 
     /**
+     * Return the TF frame ID corresponding to the local origin with the slash removed
```

Comment:
Note to self: fix this documentation. This function returns the frame *with* a slash, not without.